### DISCUSSION TITLE
Arch-go is now listed in Awesome-Go

### DISCUSSION THREAD

**Discussion by fdaines** [arch-go/arch-go#97] [Announcements]
3 months ago I sent a [PR](https://github.com/avelino/awesome-go/pull/5309) to the awesome-go project repository adding Arch-Go into the testing tools section, yesterday I received the notification that the PR was approved and merged, so then now we're listed in [https://awesome-go.com](https://awesome-go.com).

As I sent the PR 3 months ago, I added the project as `github.com/fdaines/arch-go`, but I checked this url and is redirecting to `github.com/arch-go/arch-go` (because of some kind of magic related to the project migration I've done a few weeks ago). I will create a new PR for awesome-go replacing the old url to the new one, but I think that we need to wait some time before we can see the new url in the awesome-go page.

I believe that our project will have more visibility into the community and with that, more users, new feedback and feature requests

**Comment by dadrus** [comment]
That are really great news! Incredible!
